app/src/main/java/com/example/mytask/presentation/MainActivity.kt

```diff
@@ -12,11 +17,19 @@ import com.example.mytask.data.entities.University
 import com.example.mytask.data.entities.relations.ClassroomSubjectCrossRef
 import com.example.mytask.data.entities.relations.TeacherSubjectCrossRef
 import com.example.mytask.databinding.ActivityMainBinding
+import com.example.mytask.presentation.main.MainViewModel
 import com.example.mytask.presentation.welcome.WelcomeFragment
 import kotlinx.coroutines.launch
 
 class MainActivity : AppCompatActivity() {
 
+    private  val mainViewModel = ViewModelProvider(this).get(MainActyvityViewModel::class.java)
```

Comment:
Will be better to add a lateinit modifier and init it in onCreate method. Because later you will be needed to add a viewModelFactory and it not be able here